llms/mlx_lm/utils.py

```diff
@@ -142,42 +142,40 @@ def generate(
     return tokens
 
 
-def load(path_or_hf_repo: str) -> Tuple[nn.Module, PreTrainedTokenizer]:
+def load_model(model_path: Path) -> nn.Module:
```

Comment:
Just move the `load_model` out of the `load` function in case we only want to load the model and leave it flexible for users to initialize their own tokenizer. The tokenizer is currently very variable in LLM space at the moment.